lib/Doctrine/ORM/Mapping/DiscriminatorColumn.php

```diff
@@ -51,4 +51,9 @@
      * @var string
      */
     public $columnDefinition;
+
+    /**
+     * @var bool
+     */
+    public $strict;
```

Comment:
As I mentioned before, an addition here is not to be considered unless an actual problem is being solved by it. So far, it seems like this only leads to removal of an `IN()` condition on discrimitor values when set to `false`